crates/ra_syntax/src/parsing/reparsing.rs

```diff
@@ -182,6 +193,8 @@ mod tests {
         let fully_reparsed = SourceFile::parse(&after);
         let incrementally_reparsed: Parse<SourceFile> = {
             let f = SourceFile::parse(&before);
+            // FIXME: it seems this initialization statement is unnecessary (see edit in outer scope)
+            // Investigate whether it should really be removed.
             let edit = AtomTextEdit { delete: range, insert: replace_with.to_string() };
```

Comment:
Neither `range` nor `replace_with` change, and `edit` is always passed as reference, so I'm inclined to agree that this line could probably be removed.